Leanplum-SDK/Classes/LPActionContext.h

```diff
@@ -99,6 +101,12 @@ NS_SWIFT_NAME(trackMessage(event:value:info:params:));
  */
 - (BOOL)hasMissingFiles;
 
+/**
+ * Returns a copy of the Context arguments
+ */
+- (NSDictionary *)getArgs
```

Comment:
naming getter as `getSomething` is not common for iOS, so since we have `args` as property maybe you can make it public and create getter and setter methods dynamically.